### DISCUSSION TITLE
Non-scrollable scrollbar on Chat Composer

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3515] [Q&A] [answered]
Hi Paste team,

On the Paste Chat Composer previews, I see a non-scrollable scrollbar for some reason on the example section. See screenshot below:
<img width="738" alt="Screenshot 2023-10-03 at 5 04 09 PM" src="https://github.com/twilio-labs/paste/assets/9394413/92e45fb4-0ff6-4052-b879-0b0ab685d8ce">

Is there a reason for the scrollbars?  Not sure if it's an issue on my end, but I can see it on Google Chrome, Firefox, and incognito.

**Comment by SiTaggart** [comment, marked as answer]
Hi @tonge495 

Eagle eyes! If anything, it's kind of a mistake. Switching them from `overflow: scroll` to `overflow: auto` would only show the scroll bars when necessary.

If it's only an example implementation detail, I'm not super worried. If it's baked into the component itself, I'll try and schedule a small patch fix to neaten it up.

**Reply by tonge495** [reply]
Thanks for the follow up!  Yea, the developer I'm working with said he's not sure if it's component level or not.  For what it's worth, it seems to be showing up on my side but he hasn't really seen it on his end.  We can let you know if we notice anything else with it!

**Reply by serifluous** [reply]
You can follow along the fix in PR #3543
